tests/AuthTest.php

```diff
@@ -17,10 +17,10 @@ class AuthTest extends TestCase
 {
     public function tearDown(): void
     {
-        parent::setUp();
```

Comment:
Was this a longstanding bug? I don't understand why a `setUp()` method should ever have been called during `tearDown()`.